backend/functions/lib/cosmos-humble-objects/cosmos-database-humble.ts

```diff
@@ -0,0 +1,25 @@
+import { ClientContext, Containers, CosmosClient, Database, Users } from '@azure/cosmos';
+import CosmosContainerHumble from './cosmos-container-humble';
+
+export default class CosmosDatabaseHumble {
+  private containers: Containers;
+  private database: Database;
+  private client: CosmosClient;
+  private clientContext: ClientContext;
+  private id: string;
+  private users: Users;
+
+  constructor(client: CosmosClient, id: string, context: ClientContext) {
+    this.client = client;
+    this.clientContext = context;
+    this.id = id;
+    this.database = new Database(this.client, this.id, this.clientContext);
+    this.containers = new Containers(this.database, this.clientContext);
+    this.users = new Users(this.database, this.clientContext);
```

Comment:
@jamesobrooks are we using this user value or managed ID?

Reply:
James and I removed the Users for now.